### PR TITLE
[#feat] Add example to describe Airflow DAG using yaml odahu resource…

### PR DESCRIPTION
**Pull request by vlad-tokarev**
… definition files

### CODE REVIEW

File: mlflow/sklearn/wine/airflow/dag_from_yamls/training.odahuflow.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      localPath: mlflow/sklearn/wine/wine-quality.csv
+  hyperParameters:
+    alpha: "1.0"
+  resources:
+    limits:
+      cpu: 4024m
+      memory: 4024Mi
+    requests:
+      cpu: 2024m
+      memory: 2024Mi
```
Comment (maxkochubey) [inline mlflow/sklearn/wine/airflow/dag_from_yamls/training.odahuflow.yaml]: Please consider values in a more familiar form :)
```
  resources:
    limits:
      cpu: 4
      memory: 4Gi
    requests:
      cpu: 2
      memory: 2Gi
```

**Review by maxkochubey — APPROVED** [review]

File: mlflow/sklearn/wine/airflow/dag_from_yamls/dag_from_yamls.py [hunk 2]
```diff
@@ -0,0 +1,156 @@
+from datetime import datetime
+
```
Comment (mcdoker18) [inline mlflow/sklearn/wine/airflow/dag_from_yamls/dag_from_yamls.py:2]: We need a little documentation or link to the documentation here. This doc should explain how to package the dag and yamls file and to deploy it to a composer.

File: mlflow/sklearn/wine/airflow/dag_from_yamls/dag_from_yamls.py [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+default_args = {
+    'owner': 'airflow',
+    'depends_on_past': False,
+    'start_date': datetime(2019, 9, 3),
+    'email_on_failure': False,
+    'email_on_retry': False,
+    'end_date': datetime(2099, 12, 31)
+}
+
+edi_connection_id = "odahuflow_edi"
```
Comment (mcdoker18) [inline mlflow/sklearn/wine/airflow/dag_from_yamls/dag_from_yamls.py]: `edi` -> `api`

**Review by mcdoker18 — CHANGES_REQUESTED** [review]

File: mlflow/sklearn/wine/airflow/dag_from_yamls/dag_from_yamls.py [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+        destination_object='data/',
+        project_id=gcp_project,
+        default_args=default_args
+    )
+    data_transformation = BashOperator(
+        task_id='data_transformation',
+        bash_command='echo "imagine that we transform a data"',
+        default_args=default_args
+    )
+    odahuflow_conn = GcpConnectionToLegionConnectionOperator(
```
Comment (mcdoker18) [inline mlflow/sklearn/wine/airflow/dag_from_yamls/dag_from_yamls.py]: Odahu

**Review by mcdoker18 — CHANGES_REQUESTED** [review]

File: mlflow/tensorflow/example/airflow/dag.py [hunk 5]
```diff
@@ -80,21 +80,21 @@
 with dag:
     train = TrainingOperator(
         task_id="training",
-        edi_connection_id=edi_connection_id,
+        edi_connection_id=api_connection_id,
```
Comment (mcdoker18) [inline mlflow/tensorflow/example/airflow/dag.py]: Rename the `edi_connection_id` in all files as well.

**Review by mcdoker18 — CHANGES_REQUESTED** [review]

**Review by mcdoker18 — APPROVED** [review]

**Pull request merged**